mobile/Screen/SubmissionScreen.js

```diff
@@ -1,20 +1,198 @@
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useState, useEffect } from 'react';
+import {
+  StyleSheet, TextInput, Text, View, ScrollView, Button, Image,
+} from 'react-native';
+import axios from 'axios';
+import * as ImagePicker from 'expo-image-picker';
+
+const { URL } = process.env;
```

Comment:
I'm unable to connect to the database with axios and I'm unsure if this is the reason